securedrop/bin/dev-deps

```diff
@@ -34,8 +34,8 @@ function run_x11vnc() {
 }
 
 function urandom() {
-    sudo rm /dev/random
-    sudo ln -s /dev/urandom /dev/random
+    sudo rm /dev/random ||:
+    sudo ln -s /dev/urandom /dev/random || echo "Unable to replace /dev/random"
```

Comment:
Haven't checked yet, but one issue with this is that /dev/random is blocking whereas /dev/urandom is not. This could lead to dodgy behaviour on, for example, generating a bunch of test sources and associated keys. It might be worth running loaddata.py with a heckton of sources and replies to see how it behaves compared to the docker version.

Reply:
My understanding is that on modern Linux kernels (2020+), /dev/random is mostly the same as /dev/urandom.

See https://lwn.net/Articles/808575/ and https://www.phoronix.com/scan.php?page=news_item&px=Linux-5.6-Random-Rework - the CRNG gathers entropy and blocks at boot, and then it'll no longer block at runtime.

Reply:
Said changes apply to 5.6+ apparently, our grsec kernel is currently at 5.4.* - though maybe they've been backported? This doesn't apply to the dev environment obviously, as it's going to be using the host kernel, but it's worth noting that 5.4.* is still pretty common across the board (e.g. in my debian10-based Qubes dev VM, I'm on 5.4.156). It's probably worth checking said behaviour.

Reply:
Ah yeah, I'm also on 5.4 because of Qubes 4.0. I did a bit of testing with loaddata.py+podman+`/dev/random` as-is using rough benchmarking:

3 sources + 3 journalists (default): ~8.7s, 2.9s per source+journalist pair
15 + 15: ~42.7s, ~2.85s/pair
90 + 90: ~254.6s, ~2.83s/pair

I figured out by the last run that the part that really takes time is the source GPG key generation, but it seems to be pretty constant time-wise.

Two other options besides soft failing:
* Get rid of this entirely, maybe in a separate PR, as unnecessary
* Wrap it in a check for `USE_PODMAN=1` so nothing changes for Docker users